packages/conversation/src/models/v1/mod-callback-events/capability-event/capability-event.ts

```diff
@@ -6,27 +6,13 @@ import { ConversationCallbackEvent } from '../conversation-callback-event';
  * This callback is used to deliver the results of the asynchronous capability checks.
  */
 export interface CapabilityEvent extends ConversationCallbackEvent {
-
-  /** Id of the subscribed app. */
-  app_id?: string;
-  /** Timestamp marking when the channel callback was accepted/received by the Conversation API. */
-  accepted_time?: Date;
-  /** Timestamp of the event as provided by the underlying channels. */
-  event_time?: Date;
-  /** The project ID of the app which has subscribed for the callback. */
-  project_id?: string;
-  /** Context-dependent metadata. Refer to specific callback's documentation for exact information provided. */
-  message_metadata?: string;
-  /** The value provided in field correlation_id of a send message request. */
-  correlation_id?: string;
-  /** @see CapabilityEventCapabilityNotification */
-  capability_notification?: CapabilityNotification;
   /** Name of the trigger responsible for this event. */
   trigger: 'CAPABILITY';
```

Comment:
Not directly related to PR: How did you figure out the field? From manual testing? I don't see it in oas file nor docs

Reply:
`trigger` is added when parsing the event to set a discriminator. This can then be used by the SDK user to handle the event